tsconfig.json

```diff
@@ -4,17 +4,17 @@
     "allowJs": false,
     "allowSyntheticDefaultImports": true,
     "declaration": true,
+    "declarationDir": "lib",
     "esModuleInterop": true,
     "forceConsistentCasingInFileNames": true,
     "jsx": "react",
     "lib": ["es5", "es6", "dom"],
     "moduleResolution": "node",
     "noUnusedLocals": true,
-    "outDir": "lib",
     "resolveJsonModule": true,
     "skipLibCheck": true,
     "strict": true,
     "suppressImplicitAnyIndexErrors": true,
-    "target": "es6"
+    "target": "es2018"
```

Comment:
All our supported browsers from https://go.czi.team/edu-browser-support support es2018, so this is 👍

Reply:
@ahuth @jinlee93 I think that `suppressImplicitAnyIndexErrors` may have come back in a conflict resolution. We should be good in the repo. tho. Will create a task to remove this one line